healthyregions/chicago-environment-explorer
# fix: generate overlays Legend from spreadsheet data

## Problem
After creating the spreadsheet containing the Overlays that we display on the map, we still have a lot of hard-coded bits for rendering the Legend for many of the overlays

## Approach
* feat: generate the Legend programmatically based on the information from the spreadsheet

NOTE: `Impervious Surfaces` overlay is part of another PR (see #188), and may or may not work with the new Legend at this time. Once all PRs are merged, this overlay should function the same as the others

## How to Test
1. Navigate to https://deploy-preview-192--chicago-env-explorer.netlify.app/map
2. Scroll down to Overlays
3. Expand the dropdown menu and enable some overlays: `Cooling Centers`, `Historical Redlining`, and/or `Industrial or Non-residential Areas`
    * You should see the Legends appear under the Overlay dropdown
    * You should see that Legends are listed in the order that the overlays were selected/enabled
